plugins/recruiting-impl/src/components/candidates/EditCandidate.svelte

```diff
@@ -46,20 +46,31 @@
   let prevCandidate: Candidate | undefined
   let lqCandidates: QueryUpdater<Candidate> | undefined
 
-  $: lqCandidates = client.query(lqCandidates, recruiting.class.Candidate, { _id: id }, ([first]) => {
-    const adjustedRes = {
-      ...first,
-      _id: first._id as Ref<Candidate>,
-      firstName: first.firstName ?? '',
-      lastName: first.lastName ?? '',
-      avatar: first.avatar ?? '',
-      bio: first.bio ?? '',
-      address: first.address ?? {},
-      salaryExpectation: first.salaryExpectation
-    }
+  $: lqCandidates = client.query(
+    lqCandidates,
+    recruiting.class.Candidate,
+    { _id: id },
+    async ([first]) => {
+      if (candidate !== undefined) {
+        const spaceLastViews = $spacesLastViews.get(candidate.space)
+        if (spaceLastViews !== undefined) {
+          await notificationClient.readNow(spaceLastViews, candidate._id)
+        }
+      }
+
+      candidate = first
+      prevCandidate = cloneDeep(first)
+    },
+    { limit: 1 }
+  )
 
-    candidate = adjustedRes
-    prevCandidate = cloneDeep(adjustedRes)
+  onDestroy(async () => {
+    if (candidate !== undefined) {
+      const spaceLastViews = $spacesLastViews.get(candidate.space)
```

Comment:
@BykhovDenis May be we should create some high level API for notifications to get rid of similar constructions?